test/cmdlineTests.sh

```diff
@@ -345,12 +345,17 @@ printTask "Running general commandline tests..."
 
         printTask " - ${tdir}"
 
+        if [[ $(ls -A "$tdir") == "" ]]; then
```

Comment:
```suggestion
        if [[ $(ls -A "$tdir") == "" ]]
        then
```
(also below)

Reply:
Changed.

I really hate how bulky the conditions seem to be in bash because of this though. Personally I strongly prefer version with `then` and `do` on the same line.